pkg/helmexec/exec.go

```diff
@@ -258,10 +258,6 @@ func (helm *execer) SyncRelease(context HelmContext, name, chart string, flags .
 		env["HELM_TILLER_HISTORY_MAX"] = strconv.Itoa(context.HistoryMax)
 	}
 
-	if helm.IsHelm3() && helm.postRenderer != "" {
```

Comment:
Probably we remove the `postRenderer` field out of the helmExec struct now as unused?